lrlex/src/lib/ctbuilder.rs

```diff
@@ -402,7 +402,7 @@ where
 
         outs.push_str(&format!(
             "{mod_vis} mod {mod_name} {{
-use cfgrammar::Span;
+use ::cfgrammar::Span;
```

Comment:
Ah, no what I mean is: we don't want to introduce `Span` into the namespace at all. In other words, we should kill this `use`, and wherever we reference `Span` change it to `::cfgrammar::Span` (there are some instances of this already IIRC).

Reply:
Aha, now I see should be fixed in [860575e](https://github.com/softdevteam/grmtools/pull/330/commits/860575eaf7b85a2ebf99762d66ffa9e4917597c9)